storage/badger/computation_result.go

```diff
@@ -0,0 +1,48 @@
+package badger
+
+import (
+	"github.com/dgraph-io/badger/v2"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage/badger/operation"
+)
+
+type ComputationResultUploadStatus struct {
+	db *badger.DB
+}
+
+func NewComputationResultUploadStatus(db *badger.DB) *ComputationResultUploadStatus {
+	return &ComputationResultUploadStatus{
+		db: db,
+	}
+}
+
+func (c *ComputationResultUploadStatus) Store(computationResultID flow.Identifier,
```

Comment:
Why don't key by blockID?

Reply:
i used to think about the same. the problem is on EDS, we are only able to use ExecutionDataID to retrieve stored data in which we can find BlockID, so I simply use ExecutionDataID(renamed).

Reply:
addressed - now this new status data is keyed by `blockID`.